spec/controllers/course/mail_templates_controller_spec.rb

```diff
@@ -0,0 +1,101 @@
+require 'rails_helper'
+
+RSpec.describe Course::MailTemplatesController, :type => :controller do
+  before do
+    ActsAsTenant.current_tenant = Instance.default
+    sign_in(create(:administrator))
+  end
+  let!(:course) { create(:course) }
+  let!(:existing) { create(:invitation, course: course) }
+  let(:new_template) { build(:announcement, course: course).as_json }
+
+  def expect_correct_response(redirect_path, controller_action, options = {})
```

Comment:
Assignment Branch Condition size for expect_correct_response is too high. [23.09/15]